articles/connectors/connectors-native-http.md

```diff
@@ -302,6 +302,17 @@ HTTP requests have a [timeout limit](../logic-apps/logic-apps-limits-and-config.
 
 <a name="disable-location-header-check"></a>
 
+### Retry-After  
+The http action respects the retry after with 429 status code if received one .
+the value will be considered as Seconds
```

Comment:
Delete this line as the suggestion includes this text.

```suggestion
the value will be considered as Seconds
```

Reply:
@mbarqawi: This line still appears, "the value will be considered as Seconds". Please delete.